packages/backup-restore/src/commands/projects.ts

```diff
@@ -0,0 +1,107 @@
+import fs from "fs/promises";
+import path from "path";
+
+import { exists } from "../lib/helpers";
+import { configstore, ConfigOptions } from "../config/configstore";
+import { paths } from "../config/paths";
+
+import { IServiceAccountJson } from "../lib/firebase";
+import { FsErrors } from "../lib/types";
+
+/**
+ * List project IDs of available firebase credentials
+ */
+export function listProjectCredentials(): void {
+  const projects = configstore.get(ConfigOptions.Projects);
+  console.log(projects);
+}
+
+// TODO: following two functions could be fragmented and tested better
+/**
+ * Add firebase project credentials to XDG-specific AppData location
+ */
+export async function addProjectCredentials(filePath: string): Promise<void> {
+  // Resolve path to file and check it exists
+  const currentDir = process.cwd();
+  const resolvedPath = path.resolve(currentDir, filePath);
```

Comment:
Non need to pass `currentDir` here: it's already the default. From `path.resolve` docs:
>  If after using all {from} paths still no absolute path is found, the current working directory is used as well.